### DISCUSSION TITLE
Improve create-payload-app templates

### DISCUSSION THREAD

**Discussion by denolfe** [payloadcms/payload#1602] [Feature Requests & Ideas]
The currently available templates for create-payload-app are very simplistic. There are many powerful features that these templates should be using. 

For instance, the blog template could start using autosave and versions, have more sophisticated page building patterns, add a few additional rich text elements, and more. Essentially add some essential features and settings that will make it more like the public demo.

**Comment by geminigeek** [comment]
hi,

I think a minimum front end for blog template made with  react / vue  using tailwind, can push payload a lot , the real power of headless CMS coupled with the instant realization of what can be built, its just awesome for a new user. like a joomla default template where we have a preview site button, the blocks field will shine in the template
